azkaban-common/src/main/java/azkaban/scheduler/TriggerBasedScheduleLoader.java

```diff
@@ -132,33 +129,29 @@ public void updateSchedule(Schedule s) throws ScheduleManagerException {
     }
   }
 
-  // TODO may need to add logic to filter out skip runs
   @Override
   public synchronized List<Schedule> loadSchedules()
       throws ScheduleManagerException {
-    List<Trigger> triggers = triggerManager.getTriggers(triggerSource);
-    List<Schedule> schedules = new ArrayList<Schedule>();
-    for (Trigger t : triggers) {
-      lastUpdateTime = Math.max(lastUpdateTime, t.getLastModifyTime());
-      Schedule s = triggerToSchedule(t);
-      schedules.add(s);
-      System.out.println("loaded schedule for "
-          + s.getProjectName() + " (project_ID: " + s.getProjectId() + ")");
-    }
-    return schedules;
-
+    return null;
   }
 
   private Schedule triggerToSchedule(Trigger t) throws ScheduleManagerException {
-    Condition triggerCond = t.getTriggerCondition();
-    Map<String, ConditionChecker> checkers = triggerCond.getCheckers();
-    BasicTimeChecker ck = null;
+    Map<String, ConditionChecker> checkers = t.getTriggerCondition().getCheckers();
+    BasicTimeChecker triggerTimeChecker = null;
     for (ConditionChecker checker : checkers.values()) {
       if (checker.getType().equals(BasicTimeChecker.type)) {
-        ck = (BasicTimeChecker) checker;
-        break;
+        triggerTimeChecker = (BasicTimeChecker) checker;
```

Comment:
Why do we need to break here? The logics here is confusing, and I added a todo comment to clarify.